chia/pools/pool_wallet.py

```diff
@@ -847,8 +840,16 @@ async def claim_pool_rewards(self, fee: uint64) -> TransactionRecord:
             type=uint32(TransactionType.OUTGOING_TX.value),
             name=full_spend.name(),
         )
-        await self.wallet_state_manager.add_pending_transaction(absorb_transaction)
-        return absorb_transaction
+
+        fee_tx = None
+        if fee > 0:
+            absorb_announce = Announcement(current_coin_record.coin.name(), b"$")
+            fee_tx = await self.generate_fee_transaction(fee, coin_announcements=[absorb_announce])
+            full_spend = SpendBundle.aggregate([fee_tx.spend_bundle, claim_spend])
```

Comment:
It looks like we don't do anything with full_spend here.  I think we need it to be pushed on the absorb_transaction.

Reply:
If I'm correct and the tests still pass we should add some `get_confirmed_balance` checks to make sure our wallet balance drops by the appropriate fee amount.

Reply:
Cool, I'll update that test